### ISSUE TITLE
"make setup" croaks on installation of mpy-cross

### ISSUE THREAD

**Issue by amotl** [hiveeyes/terkin-datalogger#57]
@thiasB reported that
```
make setup
```
would croak with
```
ERROR: Could not find a version that satisfies the requirement mpy-cross==1.11 (from versions: none)
ERROR: No matching distribution found for mpy-cross==1.11
```

**Comment by amotl** [comment]
He resolved it by
> Downgrade auf `pip==19.3.1` in `core.mk` before invoking `@$(pip2)` makes things work again.

**Comment by thiasB** [comment]
Actually, it's in tools/micropython.mk
fixed with #58

**Comment by amotl** [comment]
Thanks!
